### ISSUE TITLE
Image disappears when switching tabs in TabBar

### ISSUE THREAD

**Issue by Shalini-Ashokan** [dotnet/maui#22427]
### Description

If an image is loaded, and you switch tabs using the TabBar at the bottom, the image disappears from the page. This scenario occurs only in Android platform. It works fine on other platforms (Windows, iOS, and macOS)

### Steps to Reproduce

1. Create a new Maui application.
2. Load an image using stream.
3. Run the application on an Android platform.

### Link to public reproduction project repository

https://github.com/Shalini-Ashokan/AndroidImageDemo

### Version with bug

Unknown/Other

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 10

**Comment by ziomek64** [comment]
Seems like my #14471 @mattleibow

**Comment by AlleSchonWeg** [comment]
@Shalini-Ashokan
in my app (Version: 8.0.90-ci.net8.24420.2) i can't reproduce this anymore. It looks like the issue is fixed.

**Comment by ziomek64** [comment]
Its fixed and has to be closed

**Comment by rezamohamed** [comment]
I am on v8.0.93 and still have this issue where FontImageSource will intermittently not show the Glyph when you tab around in AppShell and return.

`<FontImageSource FontFamily="FontIcons" Glyph="{StaticResource Pen}" />`

**Comment by AlleSchonWeg** [comment]
> I am on v8.0.93 and still have this issue where FontImageSource will intermittently not show the Glyph when you tab around in AppShell and return.
> 
> `<FontImageSource FontFamily="FontIcons" Glyph="{StaticResource Pen}" />`

I'm using a static .cs file. Perhaps this works for your:
`
<FontImageSource FontFamily="FontIcons" Glyph="{x:Static icons:Icons.Text}" />
`
```
public static class Icons {
		public static string Text = "\ue905";
}
```

**Comment by rezamohamed** [comment]
Even if I directly hard-code the hex code, I have the issue. I posted a new issue thread about this here: https://github.com/dotnet/maui/issues/25783

**Comment by mattleibow** [comment]
Related to https://github.com/dotnet/maui/issues/14587

**Comment by kubaflo** [comment]
Closing as a duplicate of #14587. A maintainer explicitly selected #14587 as the canonical image-disappears-after-tab-navigation issue. The target is a valid issue covering the same Android tab-return image loss scenario, with no later dispute.

**Comment by ziomek64** [comment]
@kubaflo what are you talking about? Wasn't this fixed in https://github.com/dotnet/maui/issues/14471 ? This is earlier than #14587

**Comment by kubaflo** [comment]
@ziomek64 does it matter? The most important thing is that it is fixed right? :) (I used ai to close some already resolved issues)

**Comment by kubaflo** [comment]
You are right that #14471 is the earlier matching report and that the previous explanation was imprecise. A maintainer later marked this as a duplicate of #14587, while the thread also records that the original symptom was fixed. The completed closure remains appropriate, but the canonical-reference explanation should have acknowledged #14471. I also removed the stale revalidation label from this closed issue.

> Disclosure: this comment was generated by AI.
